### ISSUE TITLE
Make it possible to disable Accessibility checks

### ISSUE THREAD

**Issue by vaughandroid** [google/Accessibility-Test-Framework-for-Android#8]
As far as I can tell, it isn't possible to disable the accessibility checks once they are enabled. This makes it impossible to use for a subset of tests.

Adding a method like `AccessibilityChecks.disable()` would make it possible to add the accessibility checks to cover specific tests, rather than the whole suite.

**Comment by caseyburkhardt** [comment]
Yes, we should have exposed a `disable()` on `AccessibilityChecks`.  Will look into it.

As a workaround for now, give this a shot:
`ViewActions.removeGlobalAssertion(AccessibilityChecks.accessibilityAssertion());`

**Comment by vaughandroid** [comment]
Finally got around to trying to build a `TestRule` using this workaround. It doesn't work, since `AccessibilityChecks.checksEnabled` doesn't get reset. I've hacked it for now with reflection.

For anyone else who runs into this, it looks like the commit here fixes it in AndroidX: https://github.com/android/android-test/commit/ac49334ff84864c948c9a3ec2213db778aea5a57

**Comment by caseyburkhardt** [comment]
Resolved by [this change](https://github.com/android/android-test/commit/ac49334ff84864c948c9a3ec2213db778aea5a57).
